app/frontend/src/search/keyword.js

```diff
@@ -0,0 +1,44 @@
+/* global algoliasearch instantsearch */
+import { transform, templates, renderHits } from './hits'
```

Comment:
I've vacillated on this in the past, but I now defo come down on the side of ending lines with semicolons.  Could you add these, please?

Reply:
i actually a non semicolon man in this and day and age but far more importantly its about consistency, so semicolons is fine by me. in general there are few .es/editor config things need to be tightened up